File: scripts/fib.go
package main

import (
	"fmt"
	"time"
)

func fibonacci(c int64) int64 {
	if c < 2 {
		return c
	}

	return fibonacci(c-2) + fibonacci(c-1)
}

func main() {
	startTime := time.Now()

	resultInt := fibonacci(50)

	endTime := time.Now()

	fmt.Printf("Result: %v\n", resultInt)

	fmt.Printf("Duration: %v\n", endTime.Sub(startTime))

}

File: scripts/fib.py
import datetime
import time

def fib(n):
  if n < 2:
    return n

  return fib(n-1) + fib(n-2) 

startTime = datetime.datetime.now()

print(fib(35))

endTime = datetime.datetime.now()

print("Duration: ", endTime - startTime)

